label_studio/projects/api.py

```diff
@@ -572,6 +572,23 @@ def perform_create(self, serializer):
         return instance
 
 
+class ProjectGroundTruthTaskListAPI(ProjectTaskListAPI):
+    '''
+    Same as ProjectTaskListAPI with the exception that this API only returns tasks
+    that contain at least one ground truth annotation
+    '''
+
+    def filter_queryset(self, queryset):
+        project = generics.get_object_or_404(Project.objects.for_user(self.request.user), pk=self.kwargs.get('pk', 0))
```

Comment:
Is there some special behavior for pk=0? Are we just forcing a 404 here? (presumably bc pks start at 1)

Reply:
I think just 404 because yeah they start at 1, I copied the implementation from the existing API